dev_guide/builds.adoc

```diff
@@ -111,43 +111,49 @@ in strategy definition as presented below:
 }
 ----
 
-<1> You need to specify here an image that supports incremental builds. Note: the S2I images provided by OpenShift do not implement artifact reuse, so setting incremental to true will have no effect on BuildConfigs using those builder images.
-<2> This flag controls whether an incremental build is attempted. If the builder image does not support incremental builds, the build will still succeed but you will get a log message that the incremental build was not successful because of a missing `save-artifacts` script.
-====
+<1> You need to specify an image that supports incremental builds.
+
+NOTE: The S2I images provided by the OpenShift do not implement artifact reuse,
+so setting `incremental` to `true` will have no effect on Builds using those builder images.
```

Comment:
Having the `NOTE`s separated like this messes up the callout numbering. I'd just leave em closer to how they were, or maybe phrase it "Note that the S2I images..."